app/lib/efile/line_data.yml

```diff
@@ -590,6 +590,8 @@ NJ1040_LINE_42:
   label: '42 New Jersey Taxable Income (Subtract line 41 from line 39)'
 NJ1040_LINE_43:
   label: '43 Tax on amount on line 42 (Tax Table page 54)'
+NJ1040_LINE_51:
+  label: '51 Use Tax Due on Internet, Mail-Order, or Other Out-of-State Purchases (See instructions) If no Use Tax, enter 0.00 .'
```

Comment:
dust - extra space in `0.00 .'`  , and it looks like most of these do not end in a `.` so maybe remove that to be consistent? I'm not totally sure what these are used for so not sure how much it matters